src/app/components/Header/Header.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import type { ReactNode } from 'react';
+import Logo from '../Logo/Logo';
+import BurgerButton from '../BurgerButton/BurgerButton';
+import BackButton from '../BackButton/BackButton';
+import InputBar from '../InputBar/InputBar';
+import Typography from '../Typography/Typography';
+import styles from './Header.module.css';
+
+type HeaderProps = {
+  children?: ReactNode;
+  withLogo?: boolean;
+  withBackButton?: boolean;
+  withBurgerButton?: boolean;
+  withInputBar?: boolean;
+  type?: 'search' | 'add';
+  className?: string;
+  isHighlighted?: boolean;
+};
+
+export default function Header({
+  children,
```

Comment:
I would rename this to `text`

Reply:
Resolved in 
[52777e1](https://github.com/Seborider/plugs./pull/19/commits/52777e17633e80148885400a93b0da8223551a5e)